### ISSUE TITLE
Interpret soft prompt

### ISSUE THREAD

**Issue by treena908** [thunlp/OpenPrompt#306]
I want decode the soft prompt embedding to sequence of words. I have the output soft embedding logit from BertMaskedLm and T5ForConditionalGeneration  head. the output logits shape is (batch_Size, num_tokens,3,vacab_size). How do I do that?
